apps/oracles/eth-rpc/src/lib.rs

```diff
@@ -51,6 +60,14 @@ pub struct ResponseEthCall {
     pub id: Option<u64>,
     pub error: Option<ResponseEthCallError>,
     pub result: Option<String>,
+    #[serde(default)]
+    pub rpc_url: Option<String>,
+}
+
+#[derive(Deserialize, Debug, Clone)]
+pub struct ResponsePerFeed {
+    pub feed_id: u64,
```

Comment:
nit: should be u128 as per - https://github.com/blocksense-network/blocksense/pull/1358